docs/pipelines/kopisu.md

```diff
@@ -37,7 +37,49 @@ All three methods can be run concurrently. However, we do not yet provide any me
 
 ### Freec 
 
-TODO
+For the full list of options of Freec, please refer to the tool's [manual](http://boevalab.com/FREEC/tutorial.html)
+
+Freec is a tool that needs a few tricks and workarounds to make it work. The workarounds were successfully tested for version 10.5:
+
+1. User must provide each of the chromosomes (or contigs) of his reference in seperate fasta files. These fasta files are located in the path set in `chrFiles` field of the config. To split the contents of a fasta file into several (one for each chromosome), one can use `samtools faidx`
+
+2. Chromosome names in the `.fai` file that is set in `chrLenFile` config field, must start with the prefix `chr`. Even if the chromosome names in the reference do not start with `chr`, the tool will still work  
+
+3. `mateOrientation` should be set to 0. 
+
+**Freec config example**
+For the full list of options of Freec, please refer to the tool's [manual](http://boevalab.com/FREEC/tutorial.html)
+
+Freec is a tool that needs a few tricks and workarounds to make it work. The workarounds were successfully tested for version 10.5:
+
+1. User must provide each of the chromosomes (or contigs) of his reference in seperate fasta files. These fasta files are located in the path set in `chrFiles` field of the config. To split the contents of a fasta file into several (one for each chromosome), one can use `samtools faidx`
+
+2. Chromosome names in the `.fai` file that is set in `chrLenFile` config field, must start with the prefix `chr`. Even if the chromosome names in the reference do not start with `chr`, the tool will still work  
+
+3. `mateOrientation` should be set to 0. 
+
+
+**Freec config example**
+
+```
+kopisu: 
+  use_freec_method: true  
+  output_dir: /exports/sasc/project-200-Karoly-SV/analysis/04_kopisu
```

Comment:
Could you remove the real output dirs and replace them be fake ones or placeholders? This documentation is public.